scripts/infra-mcp/tools/get_container_tags.py

```
@@ -0,0 +1,501 @@
+                    return datetime.fromisoformat(dt_str.replace('Z', '+00:00'))
+                except Exception:
+                    return datetime.min
+
+            tag_data.sort(key=lambda x: _iso(x['last_updated']) if x['last_updated'] else datetime.min, reverse=True)
+        except requests.exceptions.RequestException as e:
+            print(f"Error querying Docker Hub: {e}", file=sys.stderr)
+            return []
+        else:
+            return tag_data
+
+    def get_registry_tags(self, registry_url: str, image_name: str, limit: int = 10, architecture: str = "linux/amd64") -> list[dict[str, Any]]:
```

Ruff (0.13.1)

Unused method argument: `limit`

(ARG002)